src/templates/dashboard-spectra/dashboard-spectra.stories.js

```diff
@@ -0,0 +1,809 @@
+		label: 'Blockquote',
+		description: 'Demo',
+		toggleLabel: 'blockquote',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '9',
+		value: 'contenttimeline',
+		icon: <ChartNoAxesGantt />,
+		label: 'Content Timeline',
+		description: 'Demo',
+		toggleLabel: 'contenttimeline',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '10',
+		value: 'googlemaps',
+		icon: <Map />,
+		label: 'Google Maps',
+		description: 'Demo',
+		toggleLabel: 'googlemaps',
+		hideSelection: false,
+		useSwitch: true,
+		bagde: (
+			<Badge
+				label={ 'New' }
+				size="xs"
+				icon={ null }
+				variant="green"
+				closable={ false }
+				className="py-0 mr-2"
+			/>
+		),
+	},
+	{
+		id: '11',
+		value: 'lottieanimation',
+		icon: <House />,
+		label: 'Lottie Animation',
+		description: 'Demo',
+		toggleLabel: 'lottieanimation',
+		hideSelection: false,
+		useSwitch: true,
+	},
+	{
+		id: '12',
+		value: 'animations',
+		icon: <PanelLeftClose />,
+		label: 'Animations',
+		description: 'Documentation',
+		toggleLabel: 'animations',
+		hideSelection: false,
+		useSwitch: true,
+		bagde: (
+			<Badge
+				label={ 'Extension' }
+				size="xs"
+				icon={ null }
+				variant="yellow"
+				closable={ false }
+				className="py-0 mr-2"
+			/>
+		),
+	},
+];
+
+const astraRadioButtonGroupData = [
+	{
+		id: '1',
+		badgeText: 'Free',
+		buttonText: 'Install',
+		svg: <Logo className="size-5" />,
+		title: 'Astra Theme',
+		description: 'Free WordPress Page Builder Plugin.',
+	},
+	{
+		id: '2',
+		badgeText: 'Free',
+		buttonText: 'Activate',
+		svg: <Logo className="size-5" />,
+		title: 'Starters Templates',
+		description: 'Build your dream website in minutes with AI.',
+	},
+	{
+		id: '3',
+		badgeText: 'Free',
+		buttonText: 'Install',
+		svg: <Logo className="size-5" />,
+		title: 'SureCart',
+		description: 'The new way to sell on WordPress.',
+	},
+	{
+		id: '4',
+		badgeText: 'Free',
+		buttonText: 'Install',
+		svg: <Logo className="size-5" />,
+		title: 'Presto Player',
+		description: 'Automate your WordPress setup.',
+	},
+];
+
+const containerRowButtons = [
+	{
+		id: '1',
+		value: 'vipsupport',
+		icon: <Headset className="size-4 m-1" />,
+		label: 'VIP Support',
+		hideSelection: true,
+		useSwitch: false,
+		bagde: (
+			<Badge
+				label={ 'PRO' }
+				icon={ null }
+				variant="inverse"
+				closable={ false }
+				size="xxs"
+			/>
+		),
+	},
+	{
+		id: '2',
+		value: 'helpcenter',
+		icon: <HelpCircle className="size-4 m-1" />,
+		label: 'Help Center',
+		hideSelection: true,
+		useSwitch: false,
+	},
+	{
+		id: '3',
+		value: 'community',
+		icon: <MessageSquare className="size-4 m-1" />,
+		label: 'Join the Community',
+		hideSelection: true,
+		useSwitch: false,
+	},
+	{
+		id: '4',
+		value: 'rateus',
+		icon: <Star className="size-4 m-1" />,
+		label: 'Rate Us',
+		hideSelection: true,
+		useSwitch: false,
+	},
+];
+
+export const SpectraDashboard = () => {
+	const radioButtonGroupData = defaultRadioButtonGroupData;
```

Comment:
you can change your object name itself. any reason to re assign. check in other places too.